profiles/logical/holo/hydra/master/default.nix

```
@@ -109,6 +115,17 @@ in
     };
   };
 
+  services.restic.backups = {
+    remotebackup = {
+      user = "root";
+      paths = [ "/var/backup/postgresql/" ];
```

Let's also back up `/etc/nixos` while at it, and infer PostgreSQL backup location:

```suggestion
      paths = [
        "/etc/nixos"
        config.services.postgresqlBackup.location
      ];
```